junifer/api/res/run_venv.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ $# -lt 2 ]; then
+    echo "This script is meant to run a command within a python environment"
+    echo "It needs at least 2 parameters."
+    echo "The first one must be the environment name."
```

Comment:
the environment path in this case i suppose, might be worth mentioning here specifically

Reply:
+1

Reply:
Updated it to make it clear.